versioned_docs/version-22.10/monitoring/basic-objects/hosts.md

```diff
@@ -62,23 +61,26 @@ To change the order of the macros, click on ![image](../../assets/configuration/
 ## Notification tab
```

Comment:
Line 46, you should go back to the line for the sentence 'To view description of the macro' and shift everything to the right as in French